tests/phpunit/test-class-wc-stripe-upe-payment-method.php

```diff
@@ -317,4 +352,23 @@ public function test_payment_methods_are_reusable_if_cart_contains_subscription(
 			}
 		}
 	}
+
+	/**
+	 * Test the type of payment token created for the user.
+	 */
+	public function test_create_payment_token_for_user() {
+		$user_id = 1;
+
+		foreach ( $this->mock_payment_methods as $payment_method_id => $payment_method ) {
+			if ( 'card' === $payment_method_id ) {
+				$card_payment_method_mock = $this->array_to_object( self::MOCK_CARD_PAYMENT_METHOD_TEMPLATE );
+				$token = $payment_method->create_payment_token_for_user( $user_id, $card_payment_method_mock );
```

Comment:
```suggestion
				$card_payment_method_mock = $this->array_to_object( self::MOCK_CARD_PAYMENT_METHOD_TEMPLATE );
				$token                    = $payment_method->create_payment_token_for_user( $user_id, $card_payment_method_mock );
```
I'm not really stoked about this one, but just pointing that once more you have made a foe out of the linter.